utils.py

```diff
@@ -23,13 +23,13 @@ def banner() -> str:
             """
 
     @staticmethod
-    def check_for_updates(CURRENT_VERSION) -> str:
+    def check_for_updates(current_version: float) -> str:
         request = str(get('https://api.github.com/repos/Xample33/Rainbow-Six-Auto-Renown-Farm/contents/.github').content)
         ver = float(search('"ver(.*?)ver",', request).group(1))
-        if CURRENT_VERSION == ver:
-            return f'This version ({CURRENT_VERSION}) is the latest version.'
+        if current_version == ver:
```

Comment:
i like to use some variable in caps, just for style

Reply:
python convention (specifically PEP8) dictate that python variables should be lowercase snake_case, and only be full caps if it's a constant.
Since it's a variable passed to the function, putting it in full caps goes against styling conventions